docs/source/blog/10k-stars-announcement.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Celebrating 10,000 GitHub Stars with the Top 10 Label Studio Features
+type: blog
+image: /images/10k-stars-blog/thumbnail.png
+order: 98
+meta_title: Celebrating 10,000 GitHub Stars with the Top 10 Label Studio Features
+meta_description: From time series to active learning to the Python SDK, check out the top 10 features on the path to 10,000 GitHub stars and growing!
+---
+
+*Thank you for your contributions, participation and support of the Label Studio platform and community!*
```

Comment:
Thank you for your contributions, participation, and support of the Label Studio platform and community!*